src/deferred.js

```diff
@@ -385,5 +394,12 @@ jQuery.extend( {
 	}
 } );
 
+function deferredTick() {
+	processQueueTimeout = undefined;
+	jQuery.each( processQueue.splice( 0 ), function( _, process ) {
```

Comment:
You're right. I was thinking it just copied without the second argument. *hits forehead*

However, that assumption you mentioned seems dangerous.